spinlock.h

```diff
@@ -1,6 +1,6 @@
 #ifdef __MINGW32__
 #include <windows.h>
-static unsigned sleep(unsigned seconds)
+inline unsigned sleep(unsigned seconds)
```

Comment:
Why this change?  We don't want to be exporting a symbol called `sleep` from the library.

Reply:
Good point, hadn't thought of that!

I was trying to work around this clang error:

`In file included from libobjc2/properties.m:10:`
`libobjc2/spinlock.h:3:17: error: static declaration of 'sleep' follows non-static declaration`
`static unsigned sleep(unsigned seconds)`
`                ^`
`/opt/local/x86_64-w64-mingw32/include/unistd.h:46:22: note: previous declaration is here`
`unsigned int __cdecl sleep (unsigned int);`

Could instead replace function implementation with:

`#define sleep(seconds)  Sleep(seconds*1000)`

What do you think?